static/js/compose.js

```diff
@@ -574,7 +574,7 @@ exports.validate = function () {
 };
 
 exports.initialize = function () {
-    $('#stream,#subject,#private_message_recipient').bind({
+    $('#stream,#subject,#private_message_recipient').on({
```

Comment:
Did you test this?  I don't know if `on()` supports this calling style.  Maybe this instead:

~~~ js
var selector = $('....');
selector.on('keyup', update_fade);
selector.on('change', update_fade);